fastlane_bot/events/utils.py

```diff
@@ -542,8 +542,9 @@ def handle_flashloan_tokens(
     List[str]
         A list of flashloan tokens to fetch data for.
     """
-    flt_symbols = get_tkn_symbols(flashloan_tokens=flashloan_tokens, tokens=tokens)
     flashloan_tokens = flashloan_tokens.split(",")
+    flashloan_tokens = [cfg.w3.to_checksum_address(x) for x in flashloan_tokens]
+    flt_symbols = get_tkn_symbols(flashloan_tokens=flashloan_tokens, tokens=tokens)
```

Comment:
Needed to change the order here else the token symbols are fetched on the checksummed addresses